ydb/core/scheme/scheme_tablecell.cpp

```diff
@@ -63,6 +63,187 @@ TOwnedCellVec::TInit TOwnedCellVec::Allocate(TOwnedCellVec::TCellVec cells) {
     };
 }
 
+namespace {
+
+    struct TCellHeader {
+        TCellHeader() = default;
+
+        TCellHeader(ui32 rawValue) : RawValue(rawValue) {}
+
+        TCellHeader(ui32 cellSize, bool isNull)
+            : RawValue(cellSize | (static_cast<ui32>(isNull) << 31ULL))
```

Comment:
You shouldn't use `ULL` on the right operand of bitwise shift, just use a plain `31`.

Reply:
Updated.

Reply:
Updated.